source/fundamentals/indexes.txt

```diff
@@ -0,0 +1,184 @@
+=======
+Indexes
+=======
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol   
+
+Overview
+--------
+
+Indexes support the efficient execution of queries in MongoDB. Without
+indexes, MongoDB must scan *every* document in a collection to find the
+documents that match each query. Collection scans are slow and can
+negatively affect the performance of your application. When an
+appropriate index exists for a query, MongoDB uses that index to limit
+the number of documents scanned, thus improving query performance. 
+
+The :manual:`indexes section </indexes>` of the MongoDB manual provides a
+detailed overview of indexes in MongoDB.
+
+Query Coverage and Performance
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When you execute a query against MongoDB, your query can include three
+parts:
+
+- query criteria that specify field(s) and value(s) you are looking for,
+- options that affect the query's execution (e.g. read concern),
+
+- (optional) projection criteria to specify the fields MongoDB should
+  return
+
+When all the fields specified in the query criteria and projection of a
+query are indexed, MongoDB returns results directly from the index
+without scanning any documents or bringing documents into memory. Queries
+that use indexes are more performant than queries that rely on
+collection scans.
+
+The MongoDB manual addresses both :manual:`query coverage
+</core/query-optimization/#read-operations-covered-query>` and
+:manual:`index intersection </core/index-intersection>` (using multiple
+indexes to fulfil a query) in more detail.
+
+Operational Considerations
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To improve query performance, you should build indexes on fields that
+appear often in queries and for all operations that involve sorted
+results. Each index that you add consumes disk space and memory
+when active so you should track index
+memory and disk usage for capacity planning. Adding indexes also has a
```

Comment:
```suggestion
memory and disk usage for capacity planning. Adding too many indexes can have a
```

Reply:
I'm unsure its worth being entirely accurate here. Technically, all indexes will have some performance impact for writes, but this is kind of a given. I think we should frame the message as "too many".